tools/docker/build.sh

```diff
@@ -27,7 +28,7 @@ main() {
         docker_args+=(-v "${PRPLMESH_PLATFORM_BASE_DIR}:${PRPLMESH_PLATFORM_BASE_DIR}")
     fi
 
-    run docker container run "${docker_args[@]}" --rm "${DOCKER_REGISTRY}prplmesh-builder" build "$@"
+    run docker container run "${docker_args[@]}" --rm "${image}" build "$@"
```

Comment:
With my other suggestion this would be:
```suggestion
    run docker container run "${docker_args[@]}" --rm "${DOCKER_REGISTRY}${image}" build "$@"
```

Reply:
I understand your suggestion, I just don't get why it is better - I see only a dissadvantage in that the user either has to use image-build.sh or always tag the built image with the DOCKER_REGISTRY.